docs/docs/reference-react-dom-node-stream.md

```diff
@@ -0,0 +1,42 @@
+---
+id: react-dom-node-stream
+title: ReactDOMNodeStream
+layout: docs
+category: Reference
+permalink: docs/react-dom-node-stream.html
+---
+
+If you use ES6 with npm, you can write `import ReactDOMNodeStream from 'react-dom/node-stream'`. If you use ES5 with npm, you can write `var ReactDOMNodeStream = require('react-dom/node-stream')`.
+
+Unlike other packages in React, `ReactDOMNodeStream` depends on a package (`stream`) that is available in Node.js but not in the browser. For this reason, there is no `<script>` tag version of `ReactDOMNodeStream`; it is only provided as a Node.js module.
+
+## Overview
+
+The `ReactDOMNodeStream` class allows you to render your components in Node.js and stream the resulting markup.
```

Comment:
Right you are! 

Should be fixed in b6f9e06, and I opened PR #10027 to fix it in the doc for `ReactDOMServer`, too.

Thanks for the help!